### ISSUE TITLE
Improvements to map displays

### ISSUE THREAD

**Issue by GilesGibson** [folklabs/openlocal#18]
Looks good as a start.

Longer term for the project we may have to have a set of categories such as :

Grot spot
Parking issue
Dangerous for pedestrians
Disused ground
Child un-friendly
empty shop

And so on….

Adding these to the map would be good. Hopefully if the project goes well we can show “improved area, no grot-spot, and so on”

How can users add a pin to the map?

When clicking on a pin, what info is shown and could it take you somewhere on the site…?

**Comment by GilesGibson** [comment]
from @djwesto 

@GilesGibson, I now see what you want to do with this tool (presumably your "Grot spot" issues are related to CLIPS etc?)
I don't think it would be a good idea for users to be able to create Locations on the map and then directly categorise them as "Grot spot" – Locations are really currently about identifying Venues and community spaces in the area. So, presently, they need to be tied to a physical postal address – not any location you can click on. I think that, once you can click to place a pin, I could see this would be a really powerful feature.
@pmackay , would you imagine we might be able to reconfigure the site so that users can click any location anywhere on the map to identify a Location? We've spoken about it before but this will take quite a lot of work, won't it? Until we've done that, I don't think this idea of @GilesGibson's will work because it seems unfair that a Child's Playgroup located on a busy corner could be tagged as "Dangerous for pedestrians", even though it's the crossing which deserves attention.
I think Locations still deserve to be able to be categorised with at least some of the site-wide categories we've developed for the other content types. Then, Giles's extra categories here would act more like an extra set of status categories which would sit on top of Locations (or Issues situated in Locations) to indicate their status. It would really help to see some actual examples here...
And I suggest that (in the same way we hope to one day be able to attach a Location to a News story or Event) perhaps we could use the Issue content type to raise these sort of issues but then attach Locations to them?
Paul, I realise it's not currently possible to draw a region on our maps but is the tech available for people to do that or is it entirely impossible to introduce this feature at a later date?

**Comment by GilesGibson** [comment]
there will be projects in any area that need additional information/assets added. CLIPS would be typical. If there could be a way of storing these assets separate from the normal ones it would be good. That way we can overlay "issue" assets to existing assets. Over time, the issue assets would not be issues any more which would be good. 

ideally the mapping also has a slider timeline so "grot spots" can slowly disappear as the project progresses.

**Comment by djwesto** [comment]
Yikes. This is getting messy. "Issue Assets" just sounds like a real opportunity to confuse people. Either an asset is an asset or it isn't. I think that certain Locations _can_ have a type of "Asset" where necessary but then an Issue that you've spotted and which you decide to raise against a location can be added to a Location later (some of those Locations might be assets but some won't be). Does that seem a little simpler?

**Comment by pmackay** [comment]
Indeed. Issue assets is an oxymoron. By issues is that like Fix My Street
stuff? Would it be better to use that?  And I find assets a bit confusing.
Any chance we could stick with resources and locations?

**Comment by djwesto** [comment]
I assumed the set of Problem types that @GilesGibson listed out in his earlier email (Grot Spots et al) were related to what he wants to do for CLIPS? Fix My Streets is great but exists on another site – presumably where they're not going to be running CLIPS. I was sort of assuming these ideas were more related to what we might do for the Tulse Hill project?

I agree that Resources and Locations should be our two main terms. I see Assets and Resources as interchangeable terms. But, if @GilesGibson sees a subtle difference I'd appreciate a sample list of each so we can see it ourselves.

**Comment by pmackay** [comment]
@GilesGibson I'm struggling with this a bit. Please could you create separate issues in the form of user stories (as a ....., I can ......., so that I can ......). This would hopefully separate out key things needed. Be very specific about CLIPS and/or other requirements if appropriate. Currently there feels like too many overlapping and slightly unclear possible ideas..

**Comment by GilesGibson** [comment]
Ok, will do
